lib/net.c

```diff
@@ -1230,7 +1300,8 @@ init_net_stage1(void *arg)
 		RTE_VERIFY(ret > 0 && ret < (int)sizeof(pool_name));
 		net_conf->gatekeeper_pktmbuf_pool[i] =
 			rte_pktmbuf_pool_create(pool_name,
-				GATEKEEPER_MBUF_SIZE, GATEKEEPER_CACHE_SIZE, 0,
+				gatekeeper_num_mbuf,
+				gatekeeper_per_core_cache_size, 0,
```

Comment:
Variables `gatekeeper_num_mbuf` and `gatekeeper_per_core_cache_size` are only used here, why don't we just use `net_conf->gatekeeper_num_mbuf` and `net_conf->gatekeeper_per_core_cache_size`, respectively, here?